cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -194,21 +229,25 @@ private void executeAction(ApplicationSpecification appSpec, WorkflowActionNode
 
     final WorkflowActionSpecification actionSpec;
     ScheduleProgramInfo actionInfo = node.getProgram();
+    final boolean isCustomAction;
```

Comment:
No need for this variable. We can use `actionInfo.getProgramType` to determine whether the action is `CUSTOM_ACTION`.